### PR TITLE
Added sort on table

### PR DESCRIPTION
**Pull request by ghost**
Added new feature based on #7

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -1,105 +1,108 @@
...
-          <div class="col-xs-12 col-md-6 col-lg-4">
-            <div class="form-group">
-              <label for="region-select">Region</label>
-              <select class="form-control selectpicker" id="region-select" data-live-search="true">
+    <div class="pricing-table">
+      <div class="row">
+        <div class="col-xs-12 col-md-6 col-lg-4">
+          <div class="form-group">
+            <label for="region-select">Region</label>
+            <select class="form-control selectpicker" id="region-select" data-live-search="true">
```
Comment (recrsn) [inline app/index.html]: Please don't add style changes with your own change. It becomes difficult to review your change

File: app/index.html [hunk 2]
```diff
@@ -1,105 +1,108 @@
...
+            <tr id="table-headers">
+              <th data-field="region">Region <span class="glyphicon glyphicon-sort"></span></th>
+              <th data-field="type">Type <span class="glyphicon glyphicon-sort"></span></th>
+              <th data-field="size">Size <span class="glyphicon glyphicon-sort"></span></th>
+              <th data-field="price">Price <span class="glyphicon glyphicon-sort"></span></th>
             </tr>
-            </thead>
-            <tbody id="pricing-data">
+          </thead>
+          <tbody id="pricing-data">
```
Comment (recrsn) [inline app/index.html]: Same, simplify this diff

File: app/scripts/main.js [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
-  let $size = $('#size-select');
+  const $data = $('#pricing-data');
+  const $region = $('#region-select');
+  const $type = $('#type-select');
+  const $size = $('#size-select');
+  let sortOrder = '';
+  let sortField = '';
 
-  let loadData = function (data) {
+  const loadData = function(data) {
```
Comment (recrsn) [inline app/scripts/main.js]: Revert these changes

**Review by recrsn — CHANGES_REQUESTED** [review]

**Comment by recrsn** [comment]
@vincealdrin While I appreciate your effort, I ask you to simplify this PR to include your changes only. With your changes mixed with other changes, the diff becomes too dirty to review.

Fixing it according to the linter config is another task.

**Comment by ghost** [comment]
@agathver Okay thanks, I'll work on the changes
